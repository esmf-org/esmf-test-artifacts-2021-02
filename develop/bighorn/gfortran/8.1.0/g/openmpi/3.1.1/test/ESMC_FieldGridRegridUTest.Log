Tue Jan 19 05:37:39 MST 2021
