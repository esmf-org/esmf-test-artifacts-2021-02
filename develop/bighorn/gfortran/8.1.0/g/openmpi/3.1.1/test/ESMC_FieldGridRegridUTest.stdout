[bighorn.cgd.ucar.edu:19545] OPAL ERROR: Error in file pmix2x.c at line 326
[bighorn.cgd.ucar.edu:19545] OPAL ERROR: Error in file pmix2x.c at line 326
[bighorn.cgd.ucar.edu:19545] OPAL ERROR: Error in file pmix2x.c at line 326
[bighorn:19545] *** Process received signal ***
[bighorn:19545] Signal: Segmentation fault (11)
[bighorn:19545] Signal code: Invalid permissions (2)
[bighorn:19545] Failing at address: 0x7f35e8021108
[bighorn:19545] [ 0] /lib64/libpthread.so.0(+0xf630)[0x7f35f721a630]
[bighorn:19545] [ 1] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/openmpi/mca_pmix_pmix2x.so(pmix2x_value_unload+0x0)[0x7f35f495dae0]
[bighorn:19545] [ 2] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/openmpi/mca_pmix_pmix2x.so(pmix2x_event_hdlr+0x2dd)[0x7f35f495e83d]
[bighorn:19545] [ 3] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/openmpi/mca_pmix_pmix2x.so(pmix_invoke_local_event_hdlr+0x328)[0x7f35f4978f38]
[bighorn:19545] [ 4] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/openmpi/mca_pmix_pmix2x.so(+0x3c9ea)[0x7f35f497d9ea]
[bighorn:19545] [ 5] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/openmpi/mca_pmix_pmix2x.so(+0x3e67e)[0x7f35f497f67e]
[bighorn:19545] [ 6] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/openmpi/mca_pmix_pmix2x.so(pmix_ptl_base_process_msg+0x1cd)[0x7f35f49e845d]
[bighorn:19545] [ 7] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/libopen-pal.so.40(opal_libevent2022_event_base_loop+0x759)[0x7f35f6904509]
[bighorn:19545] [ 8] /project/esmf/LIBS/gcc-8.1/openmpi-3.1.1/lib/openmpi/mca_pmix_pmix2x.so(+0x80f5e)[0x7f35f49c1f5e]
[bighorn:19545] [ 9] /lib64/libpthread.so.0(+0x7ea5)[0x7f35f7212ea5]
[bighorn:19545] [10] /lib64/libc.so.6(clone+0x6d)[0x7f35f6f3b96d]
[bighorn:19545] *** End of error message ***
-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
-------------------------------------------------------
--------------------------------------------------------------------------
mpirun noticed that process rank 0 with PID 0 on node bighorn exited on signal 11 (Segmentation fault).
--------------------------------------------------------------------------
Tue Jan 19 05:37:39 MST 2021
